Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/patientdiscovery/inbound/deferred/request/StandardInboundPatientDiscoveryDeferredRequestTest.java

```diff
@@ -59,11 +62,13 @@
  */
 public class StandardInboundPatientDiscoveryDeferredRequestTest {
 
+    private final PatientDiscoveryDeferredRequestAuditLogger auditLogger = mock(PatientDiscoveryDeferredRequestAuditLogger.class);
```

Comment:
If you need to make another commit, can you wrap this line?  (Don't make a commit just for this)